### ISSUE TITLE
Progranulin low in plasma

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10918]
**Preferred term label:**
Progranulin low in plasma

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ceroid lipofuscinosis 11: https://omim.org/entry/614706
Frontotemporal dementia-2: https://omim.org/entry/607485

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001187
- label: Reduced circulating progranulin concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: 
- def: The concentration of progranulin in the blood circulation is below the lower limit of normal.
- comment: Progranulin, a secreted glycoprotein, maintains cerebrocortical viability. It has been identified as an important factor controlling neural function while also suppressing neuroinflammation and playing roles in neuronal lysosome function. Loss of one allele of GRN, the gene encoding progranulin, leads to the development of frontotemporal dementia, and loss of both alleles leads to the development of neuronal ceroid lipofuscinosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10918
- synonyms:
